### PR TITLE
AWS_Cloudtrail_pack

### PR DESCRIPTION
**Pull request by sfc-gh-srathod**
Please review the aws_cloudtrail sql pack

### CODE REVIEW

File: packs/AWS_CloudTrail.sql [hunk 1]
```diff
@@ -0,0 +1,201 @@
+-- varmap:
```
Comment (anfedorov) [inline packs/AWS_CloudTrail.sql:1]: please update header to include params definition

File: packs/AWS_CloudTrail.sql [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+--   stage: 'data.{base_name}_STAGE'
+--   staging_table: 'data.{base_name}_STAGING'
+--   landing_table: 'data.{base_name}_CONNECTION'
+--   pipe = 'data.{base_name}_PIPE'
+--   task: 'data.{base_name}_TASK'
+--   stream: 'data.{base_name}_STREAM'
+--   storage_integration: 'data.{base_name}_STORAGE_INTEGRATION'
+
+
+CREATE SEQUENCE IF NOT EXISTS LANDING_TABLE_SEQ START 1 INCREMENT 1;
```
Comment (anfedorov) [inline packs/AWS_CloudTrail.sql:12]: let's keep the casing consistent -- we want our names to be lower case, so `LANDING_TABLE_SEQ` should be, as well, to separate it from the language constructs

File: packs/AWS_CloudTrail.sql [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+    ON {staging_table}
+;
+
+
+
+--Pipe
+CREATE OR REPLACE PIPE {pipe}
+    AUTO_INGEST = TRUE
+    AS
+    COPY INTO {staging_table(v)}
```
Comment (anfedorov) [inline packs/AWS_CloudTrail.sql:101]: this is not a valid param

**Review by anfedorov — CHANGES_REQUESTED** [review]
good start, but there's a few pieces left before we can test this

**Comment by sfc-gh-afedorov** [comment]
now in https://github.com/snowflakedb/snowsql-pack/tree/main/aws_cloudtrail

**Pull request closed without merging**